testenv/FakeIdp.go

```diff
@@ -89,7 +89,7 @@ func HostFakeIdp() {
 }
 
 // SubjectIDToToken creates a new jwt token for a given SubjectID
-func SubjectIDToToken(subject string) string {
+func SubjectIDToToken(subject string, orgID string, isOrgAdmin bool) string {
```

Comment:
Maybe CreateToken or something since it's not a conversion from SubjectID anymore?